Fix 182: changed reverb attribute codes, labels and a little housekee…

zztimur opened the pull request:
Fix for issue #182.

- Attribute id prefixes changed from rev_ to reverb_
- Attribute labels are updated from Rev* to Reverb*
- Some basic clean up of setup scripts
- Code adjustments due to rev->reverb modifications

skwp commented:
Awesome thanks for the contribution!

skwp commented on app/code/community/Reverb/ReverbSync/sql/reverbsync_setup/mysql4-upgrade-0.1.12-0.1.13.php:
```diff
@@ -0,0 +1,46 @@
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
```
Can we please exclude licensing info from the files? The entire project will be licensed under Apache 2.0

zztimur replied:
Removed license info

> On Feb 6, 2016, at 2:06 PM, Yan Pritzker <notifications@github.com> wrote:
> 
> In app/code/community/Reverb/ReverbSync/sql/reverbsync_setup/mysql4-upgrade-0.1.12-0.1.13.php:
> 
> > @@ -0,0 +1,46 @@
> > +<?php
> > +/**
> > + * Reverb
> > + *
> > + * NOTICE OF LICENSE
> > + *
> > + * This source file is subject to the Open Software License (OSL 3.0)
> > + * that is bundled with this package in the file LICENSE.txt.
> > + * It is also available through the world-wide-web at this URL:
> > + * http://opensource.org/licenses/osl-3.0.php
> Can we please exclude licensing info from the files? The entire project will be licensed under Apache 2.0
> 
> —
> Reply to this email directly or view it on GitHub.
>

skwp commented on app/code/community/Reverb/ReverbSync/Block/Adminhtml/Catalog/Product/Tab.php:
```diff
@@ -55,22 +55,22 @@ public function getTargetProductId() {
         try {
             $productId = Mage::registry('current_product') -> getId();
             $targetProductId = Mage::getModel('catalog/product') -> load($productId);
-            $revProductId = $targetProductId -> getRevProductId();
```
this variable didn't get renamed

skwp commented on app/code/community/Reverb/ReverbSync/Block/Adminhtml/Catalog/Product/Tab.php:
```diff
@@ -55,22 +55,22 @@ public function getTargetProductId() {
             $productId = Mage::registry('current_product') -> getId();
             $targetProductId = Mage::getModel('catalog/product') -> load($productId);
-            $revProductUrl = $targetProductId -> getRevProductUrl();
+            $revProductUrl = $targetProductId -> getReverbProductUrl();
```
same rename problem here I think

skwp commented:
Thanks for the contrib, it seems we have some merge conflicts now though. Also how did you test this? It seems to me there are two possible bugs there with the variable not renamed inside getTargetProductId and getTargetProductUrl? Thanks

zztimur commented:
Hey @skwp,
I fixed the stuff you pointed out. Thanks.

I tested after I deleted the module, completely dropped reverb tables, removed all resource references.

What's the conflict?

skwp commented:
not sure github is showing a merge conflict. let me check it out

skwp commented:
Merged my own branch of thsi thanks
